test/test.cc

```diff
@@ -21,8 +22,39 @@
 
 #include "microtar.h"
 
+namespace valhalla {
```

Comment:
did we really need to add this additional namespace. im going to guess that it causes a lot of the diffs in the tests. was the point to remove a potential namespace collision/merging with other libraries since its a very common name (`test`)?

Reply:
or maybe it doesnt make lots of diffs because most tests say `using namespace valhalla;`?

Reply:
it was an attempt to minimize changes, but better to remove this namespace.